pypit/armasters.py

```diff
@@ -187,6 +190,24 @@ def save_masters(slf, det, setup):
                            frametype='slit profile')
 
 
+def save_sensfunc(slf, setup):
+    """ Make YAML friendly and write to disk
+    Separate routine as this process is detector independent
+    
+    Parameters
+    ----------
+    slf
+    setup : str
+    """
+    # Sensitivity Function
+    if 'sensfunc' + settings.argflag['reduce']['masters']['setup'][0] not in settings.argflag['reduce']['masters']['loaded']:
```

Comment:
Not a problem/bug, but I'm not sure why the master sensitivity function is saved only with the 0th index of settings.argflag['reduce']['masters']['setup'] attached to the end of the file name, versus the entire thing, like the other Master frames (so, just 'A', instead of 'A_01_aa')?